packages/extractor/src/extractors/SpotifyExtractor.ts

```diff
@@ -50,10 +50,53 @@ export class SpotifyExtractor extends BridgedExtractor<SpotifyExtractorInit> {
     }
 
     public async getRelatedTracks(track: Track) {
-        return await this.handle(track.author || track.title, {
-            type: QueryType.SPOTIFY_SEARCH,
-            requestedBy: track.requestedBy
+        const useSpotifyApi = true;
```

Comment:
why is this value hardcoded?

Reply:
I think it will be better to first attempt getting related tracks using spotify api and if response is null, we fall back to default search method.

Reply:
I agree we can fall back to old behaviour if response is null.

The value is hardcoded just to easier compare new/old behaviour during development, it will of course be removed.